src/main/scala/com/cognite/spark/datasource/TimeSeriesRelation.scala

```diff
@@ -82,18 +120,62 @@ class TimeSeriesRelation(config: RelationConfig)(val sqlContext: SQLContext)
     df.foreachPartition(rows => {
       implicit val contextShift: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
       val batches = rows.grouped(batchSize).toVector
-      batches.parTraverse(postRows).unsafeRunSync()
+      batches.parTraverse(updateRowsZeroSix).unsafeRunSync()
     })
 
-  private def postRows(rows: Seq[Row]): IO[Unit] = {
-    val timeSeriesItems = rows.map(r => fromRow[PostTimeSeriesItem](r))
-    post(config.apiKey, baseTimeSeriesUrl(config.project), timeSeriesItems, maxRetries)
-      .map(item => {
-        if (config.collectMetrics) {
-          timeSeriesCreated.inc(rows.length)
+  private def updateRowsZeroSix(rows: Seq[Row]): IO[Unit] = {
```

Comment:
I don't think we need to call it out that it's "zero six" unless we also have something that is not "zero six" as well.

Rename it to "updateTimeSeries" or something and do the `rows.map(r => fromRow[PostTimeSeriesItem](r))` at the caller site: that's the only thing rows is being used for in this method so it's better specify that it's about PostTimeSeriesItem in the method signature.